docs/troubleshoot/certstore/cert-invalid-tsg.md

```diff
@@ -0,0 +1,21 @@
+## Troubleshoot Certificate Store Errors
+
+Please use ```kubectl get``` or ```kubectl describe``` command to retrieve the error.
+```bash
+kubectl get certificatestores.config.ratify.deislabs.io
+```
+###  CERT_INVALID
+This error is returned when Ratify fails to parse certificate fetched from Certifiate Store.
+
+#### Scenario 1
+Brieferror:       failed to get certificates fro...   
```

Comment:
not a blocker but I think we had discussed at one point adding some guidance on where these error message would be? like brieferror would be from the describe command etc.

Reply:
Ah, yes, I ve added to the description, "Please use ```kubectl get``` or ```kubectl describe``` command to retrieve the error."